src/RFC-0122_Burning.md

```diff
@@ -115,6 +115,27 @@ burned\_total = \sum_j\mathrm{C_{j}}  \\\\
 \tag{1}
 $$
 
+### Creation of burned output
+
+When creating the output, we need to be able to generate a Proof of Burn for the burned output. 
+In the header, we already commit to each and every [UTXO] ever created. There already is a proof of the existence
+for this burned output, but this proof is only to the complete output and not just the commitment. 
+
+To get around this, we enforce default values for most of the output except the version and commitment field. 
+
+And the output, when burned, MUST have the following field values
```

Comment:
I don't like these requirements. It is limiting because other chains might also want to use this functionality, and they might need a specific metadata, or script

Reply:
hmm.. for example, if you want to indicate on the second layer which side-chain the burned funds should be redeemed on; or provide a proof of ownership certificate in a script that can be picked up on the 2nd layer.

I think there are several use cases that we'd like to leverage with the metadata and script fields.

Reply:
mmm, okay, I have been convinced that we should not enforce anything here.